\cleardoublepage
% ==============================================================
\chapter{Publications}
% ==============================================================

The following publications are presented in this thesis:

\begin{enumerate}
\item 
{\bf Proteomics to go: Proteomatic enables the user-friendly creation of versatile MS/MS data evaluation workflows.}

Michael Specht, Sebastian Kuhlgert, Christian Fufezan and Michael Hippler

Bioinformatics (2011) 27 (8): 1183-1184; doi: \href{http://dx.doi.org/10.1093/bioinformatics/btr081}{10.1093/bioinformatics/btr081}.

\item
{\bf Characterizing the anaerobic response of {\em Chlamydomonas reinhardtii} by quantitative proteomics.}

Mia Terashima, Michael Specht, Bianca Naumann-Busch and Michael Hippler

Mol Cell Proteomics, 9(7), 2010: 1514-32; doi: \href{http://dx.doi.org/10.1074/mcp.M900421-MCP200}{10.1074/mcp.M900421-MCP200}.

\item
{\bf The chloroplast proteome: A survey from the {\em Chlamydomonas reinhardtii} perspective with a focus on distinctive features.}

Mia Terashima, Michael Specht and Michael Hippler

Current Genetics (2011), in press.

\item
{\bf Concerted action of the new Genomic Peptide Finder and AUGUSTUS allows for automated proteogenomic annotation of the {\em Chlamydomonas reinhardtii} genome.}

Michael Specht, Mario Stanke, Mia Terashima, Bianca Naumann-Busch, Ingrid Janßen, Ricarda H\"ohner, Erik F.~Y.~Hom, Chun Liang and Michael Hippler

Proteomics (2011), in press; doi: \href{http://dx.doi.org/10.1002/pmic.201000621}{10.1002/pmic.201000621}.

% \item
% {\bf T. oceanica paper with Markus Lommer}
% 
\item
{\bf p3d -- Python module for structural bioinformatics}

Christian Fufezan and Michael Specht

BMC Bioinformatics (2009) 10:258; doi: \href{http://dx.doi.org/10.1186/1471-2105-10-258}{10.1186/1471-2105-10-258}.

\end{enumerate}

% --------------------------------------------------------------
\cleardoublepage
\section{Proteomics to go: Proteomatic enables the user-friendly creation of versatile MS/MS data evaluation workflows}
% \markboth{Proteomics to go: Proteomatic enables the user-friendly creation of versatile MS/MS data evaluation workflows}{Proteomics to go: Proteomatic enables the user-friendly creation of versatile MS/MS data evaluation workflows}
% \addcontentsline{toc}{section}{Proteomics to go: Proteomatic enables the user-friendly creation of versatile MS/MS data evaluation workflows}

Michael Specht, Sebastian Kuhlgert, Christian Fufezan and Michael Hippler

Bioinformatics (2011) 27 (8): 1183-1184; doi: \href{http://dx.doi.org/10.1093/bioinformatics/btr081}{10.1093/bioinformatics/btr081}.

\subsection*{Synopsis}

This manuscript presents Proteomatic, a free and platform-independent data 
evaluation pipeline for the processing of mass spectrometric data. 
Proteomatic provides an accessible interface to external tools for protein
identification and quantitation.
Furthermore, a comprehensive set of helper scripts implementing auxiliary
functionality is provided.
Proteomatic provides the data evaluation framework which has been used for
data analysis in manuscripts 2 -- 4.

\subsection*{Contributions}

\begin{itemize}
\item design and implementation of the software system (Proteomatic)
\item manuscript writing
\item figure creation
\end{itemize}

% \cleardoublepage
\includepublication{publications/proteomatic-2011.pdf}

% --------------------------------------------------------------
\cleardoublepage
\section{Characterizing the anaerobic response of {\em Chlamydomonas reinhardtii} by quantitative proteomics}
% \markboth{Characterizing the anaerobic response of {\em Chlamydomonas reinhardtii} by quantitative proteomics}{Characterizing the anaerobic response of {\em Chlamydomonas reinhardtii} by quantitative proteomics}
% \addcontentsline{toc}{section}{Characterizing the anaerobic response of {\em Chlamydomonas reinhardtii} by quantitative proteomics}
% --------------------------------------------------------------

Mia Terashima, Michael Specht, Bianca Naumann-Busch and Michael Hippler

Mol Cell Proteomics, 9(7), 2010: 1514-32; doi: \href{http://dx.doi.org/10.1074/mcp.M900421-MCP200}{10.1074/mcp.M900421-MCP200}.

\subsection*{Synopsis}

In this manuscript, the chloroplast proteome of \cre~is experimentally 
determined, resulting in a core chloroplast proteome of 606 proteins and
an additional candidate proteome consisting of 289 proteins.
Furthemore, SILAC-based protein quantitation is performed to elucidate 
the anaerobic response of the \cre~chloroplast.
In order to evaluate the quantitative data obtained in the SILAC experiments,
qTrace has been developed.
Proteomatic has been employed to conduct data analysis.

\subsection*{Contributions}

\begin{itemize}
\item design and implementation of the quantitation software (qTrace)
\item data evaluation
\item manuscript writing
\item figure creation
\end{itemize}

% \cleardoublepage
\includepublication{publications/terashima-2010.pdf}

% --------------------------------------------------------------
\cleardoublepage
\section{The chloroplast proteome: A survey from the {\em Chlamydomonas reinhardtii} perspective with a focus on distinctive features}
% \markboth{The chloroplast proteome: A concise survey form the {\em Chlamydomonas reinhardtii} perspective}{The chloroplast proteome: A concise survey form the {\em Chlamydomonas reinhardtii} perspective}
% \addcontentsline{toc}{section}{The chloroplast proteome: A concise survey form the {\em Chlamydomonas reinhardtii} perspective}
% --------------------------------------------------------------

Mia Terashima, Michael Specht and Michael Hippler

Current Genetics (2011), in press.

\subsection*{Synopsis}

This manuscript extends the previously characterized chloroplast proteome
of \cre~(see manuscript 2) by adding 101 proteins and discusses the various 
pathways occurring in the organelle, focusing on the features which are 
distinctive for \cre.
A BLAST search of the extended chloroplast proteome of \cre, comprising 996
proteins, has been carried out using the NCBI non-redundant database in order
to determine the distinctiveness of the proteome in respect to higher plants
and bacteria.

\subsection*{Contributions}

\begin{itemize}
\item data evaluation (BLAST analysis)
\item manuscript writing
\item figure creation
\end{itemize}

% \cleardoublepage
\includepublication{publications/terashima-2011-ms.pdf}

% --------------------------------------------------------------
\cleardoublepage
\section{Concerted action of the new Genomic Peptide Finder and AUGUSTUS allows for automated proteogenomic annotation of the {\em Chlamydomonas reinhardtii} genome}
% \markboth{Concerted action of the new Genomic Peptide Finder and AUGUSTUS allows for automated proteogenomic annotation of the {\em Chlamydomonas reinhardtii} genome}{Concerted action of the new Genomic Peptide Finder and AUGUSTUS allows for automated proteogenomic annotation of the {\em Chlamydomonas reinhardtii} genome}
% \addcontentsline{toc}{section}{Concerted action of the new Genomic Peptide Finder and AUGUSTUS allows for automated proteogenomic annotation of the {\em Chlamydomonas reinhardtii} genome}
% --------------------------------------------------------------

Michael Specht, Mario Stanke, Mia Terashima, Bianca Naumann-Busch, Ingrid Janßen, Ricarda H\"ohner, Erik F.~Y.~Hom, Chun Liang and Michael Hippler

Proteomics (2011), in press; doi: \href{http://dx.doi.org/10.1002/pmic.201000621}{10.1002/pmic.201000621}.

\label{paper:gpf}

\subsection*{Synopsis}

This manuscript presents a re-designed version of the Genomic Peptide Finder
originally published by \citeauthor{Allmer2004} 
In addition, a method for the statistical assessment of GPF peptides is
introduced which allows for confident identification of MS/MS scans in a
way that is unbiased towards predicted gene models.
Finally, the GPF peptides resulting from various measurements of \cre~are used 
for the proteogenomic annotation of the green alga via AUGUSTUS.

\subsection*{Contributions}

\begin{itemize}
\item re-design and re-implementation of the software (GPF)
\item data evaluation
\item manuscript writing
\item figure creation
\end{itemize}

\cleardoublepage
\includepublication{publications/gpf-2011.pdf}

% % --------------------------------------------------------------
% \cleardoublepage
% \section{T.~oceanica paper with Markus Lommer}
% % \markboth{T.~oceanica paper with Markus Lommer}{T.~oceanica paper with Markus Lommer}
% % \addcontentsline{toc}{section}{T.~oceanica paper with Markus Lommer}
% % --------------------------------------------------------------
% 
% \subsection*{Contributions}
% 
% \begin{itemize}
% \item what was it, then?
% \end{itemize}
% 
% % \includepublication{publications/lommer-2011.pdf}

% --------------------------------------------------------------
\cleardoublepage
\section{p3d -- Python module for structural bioinformatics}
% \markboth{p3d -- Python module for structural bioinformatics}{p3d -- Python module for structural bioinformatics}
% \addcontentsline{toc}{section}{p3d -- Python module for structural bioinformatics}
% --------------------------------------------------------------

Christian Fufezan and Michael Specht

BMC Bioinformatics (2009) 10:258; doi: \href{http://dx.doi.org/10.1186/1471-2105-10-258}{10.1186/1471-2105-10-258}.

\subsection*{Synopsis}

Although this manuscript is not directly related to mass spectrometry-based
proteomics, it demonstrates the conceptual flexibility of Proteomatic.
p3d is a Python library for the analysis of three-dimensional structural
data stored in PDB files.
One aspect covered by the library is the extraction of entries from a PDB file
according to a user-defined query such as `protein and within 3 of resname 
HEM'\footnote{http://p3d.fufezan.net/index.php?title=PDBextract}.
While p3d is inherently command-line based, the functionality could be
integrated into Proteomatic easily and could thus be made accessible to users 
who are unfamiliar with the command-line interface.

\subsection*{Contributions}

\begin{itemize}
\item implementation of the query parser module
\item speed optimization of spatial queries
\end{itemize}

% \cleardoublepage
\includepublication{publications/fufezan-2009.pdf}
