\cleardoublepage
% ==============================================================
\chapter{Publications}
% ==============================================================

The following publications are presented in this thesis:

\begin{enumerate}
\item 
{\bf Proteomics to go: Proteomatic enables the user-friendly creation of versatile MS/MS data evaluation workflows.}

Michael Specht, Sebastian Kuhlgert, Christian Fufezan and Michael Hippler

Bioinformatics (2011) 27 (8): 1183-1184; doi: \href{http://dx.doi.org/10.1093/bioinformatics/btr081}{10.1093/bioinformatics/btr081}.

\item
{\bf Characterizing the anaerobic response of Chlamydomonas reinhardtii by quantitative proteomics.}

Mia Terashima, Michael Specht, Bianca Naumann-Busch and Michael Hippler

Mol Cell Proteomics, 9(7), 2010: 1514-32; doi: \href{http://dx.doi.org/10.1074/mcp.M900421-MCP200}{10.1074/mcp.M900421-MCP200}.

\item
{\bf The chloroplast proteome: A survey from the {\em Chlamydomonas reinhardtii} perspective with a focus on distinctive features.}

Mia Terashima, Michael Specht and Michael Hippler

Current Genetics (2011), in press.

\item
{\bf Concerted action of the new Genomic Peptide Finder and AUGUSTUS allows for automated proteogenomic annotation of the {\em Chlamydomonas reinhardtii} genome.}

Michael Specht, Mario Stanke, Mia Terashima, Bianca Naumann-Busch, Ingrid Janßen, Ricarda H\"ohner, Erik F.~Y.~Hom, Chun Liang and Michael Hippler

Proteomics (2011), in press; doi: \href{http://dx.doi.org/10.1002/pmic.201000621}{10.1002/pmic.201000621}.

% \item
% {\bf T. oceanica paper with Markus Lommer}
% 
\item
{\bf p3d -- Python module for structural bioinformatics}

Christian Fufezan and Michael Specht

BMC Bioinformatics (2009) 10:258; doi: \href{http://dx.doi.org/10.1186/1471-2105-10-258}{10.1186/1471-2105-10-258}.

\end{enumerate}

% --------------------------------------------------------------
\cleardoublepage
\section{Proteomics to go: Proteomatic enables the user-friendly creation of versatile MS/MS data evaluation workflows}
% \markboth{Proteomics to go: Proteomatic enables the user-friendly creation of versatile MS/MS data evaluation workflows}{Proteomics to go: Proteomatic enables the user-friendly creation of versatile MS/MS data evaluation workflows}
% \addcontentsline{toc}{section}{Proteomics to go: Proteomatic enables the user-friendly creation of versatile MS/MS data evaluation workflows}

Michael Specht, Sebastian Kuhlgert, Christian Fufezan and Michael Hippler

Bioinformatics (2011) 27 (8): 1183-1184; doi: \href{http://dx.doi.org/10.1093/bioinformatics/btr081}{10.1093/bioinformatics/btr081}.

\subsection*{Contributions}

\begin{itemize}
\item design and implementation of the software system (Proteomatic)
\item manuscript writing
\item figure creation
\end{itemize}

% \cleardoublepage
\includepublication{publications/proteomatic-2011.pdf}

% --------------------------------------------------------------
\cleardoublepage
\section{Characterizing the anaerobic response of {\em Chlamydomonas reinhardtii} by quantitative proteomics}
% \markboth{Characterizing the anaerobic response of {\em Chlamydomonas reinhardtii} by quantitative proteomics}{Characterizing the anaerobic response of {\em Chlamydomonas reinhardtii} by quantitative proteomics}
% \addcontentsline{toc}{section}{Characterizing the anaerobic response of {\em Chlamydomonas reinhardtii} by quantitative proteomics}
% --------------------------------------------------------------

Mia Terashima, Michael Specht, Bianca Naumann-Busch and Michael Hippler

Mol Cell Proteomics, 9(7), 2010: 1514-32; doi: \href{http://dx.doi.org/10.1074/mcp.M900421-MCP200}{10.1074/mcp.M900421-MCP200}.

\subsection*{Contributions}

\begin{itemize}
\item design and implementation of the quantitation software (qTrace)
\item data evaluation
\item manuscript writing
\item figure creation
\end{itemize}

% \cleardoublepage
\includepublication{publications/terashima-2010.pdf}

% --------------------------------------------------------------
\cleardoublepage
\section{The chloroplast proteome: A survey from the {\em Chlamydomonas reinhardtii} perspective with a focus on distinctive features}
% \markboth{The chloroplast proteome: A concise survey form the {\em Chlamydomonas reinhardtii} perspective}{The chloroplast proteome: A concise survey form the {\em Chlamydomonas reinhardtii} perspective}
% \addcontentsline{toc}{section}{The chloroplast proteome: A concise survey form the {\em Chlamydomonas reinhardtii} perspective}
% --------------------------------------------------------------

Mia Terashima, Michael Specht and Michael Hippler

Current Genetics (2011), in press.

\subsection*{Contributions}

\begin{itemize}
\item data evaluation (BLAST analysis)
\item manuscript writing
\item figure creation
\end{itemize}

% \cleardoublepage
\includepublication{publications/terashima-2011-ms.pdf}

% --------------------------------------------------------------
\cleardoublepage
\section{Concerted action of the new Genomic Peptide Finder and AUGUSTUS allows for automated proteogenomic annotation of the {\em Chlamydomonas reinhardtii} genome}
% \markboth{Concerted action of the new Genomic Peptide Finder and AUGUSTUS allows for automated proteogenomic annotation of the {\em Chlamydomonas reinhardtii} genome}{Concerted action of the new Genomic Peptide Finder and AUGUSTUS allows for automated proteogenomic annotation of the {\em Chlamydomonas reinhardtii} genome}
% \addcontentsline{toc}{section}{Concerted action of the new Genomic Peptide Finder and AUGUSTUS allows for automated proteogenomic annotation of the {\em Chlamydomonas reinhardtii} genome}
% --------------------------------------------------------------

Michael Specht, Mario Stanke, Mia Terashima, Bianca Naumann-Busch, Ingrid Janßen, Ricarda H\"ohner, Erik F.~Y.~Hom, Chun Liang and Michael Hippler

Proteomics (2011), in press; doi: \href{http://dx.doi.org/10.1002/pmic.201000621}{10.1002/pmic.201000621}.

\label{paper:gpf}

\subsection*{Contributions}

\begin{itemize}
\item re-design and re-implementation of the software (GPF)
\item data evaluation
\item manuscript writing
\item figure creation
\end{itemize}

\cleardoublepage
\includepublication{publications/gpf-2011.pdf}

% % --------------------------------------------------------------
% \cleardoublepage
% \section{T.~oceanica paper with Markus Lommer}
% % \markboth{T.~oceanica paper with Markus Lommer}{T.~oceanica paper with Markus Lommer}
% % \addcontentsline{toc}{section}{T.~oceanica paper with Markus Lommer}
% % --------------------------------------------------------------
% 
% \subsection*{Contributions}
% 
% \begin{itemize}
% \item what was it, then?
% \end{itemize}
% 
% % \includepublication{publications/lommer-2011.pdf}

% --------------------------------------------------------------
\cleardoublepage
\section{p3d -- Python module for structural bioinformatics}
% \markboth{p3d -- Python module for structural bioinformatics}{p3d -- Python module for structural bioinformatics}
% \addcontentsline{toc}{section}{p3d -- Python module for structural bioinformatics}
% --------------------------------------------------------------

Christian Fufezan and Michael Specht

BMC Bioinformatics (2009) 10:258; doi: \href{http://dx.doi.org/10.1186/1471-2105-10-258}{10.1186/1471-2105-10-258}.

\subsection*{Contributions}

\begin{itemize}
\item implementation of the query parser module
\item speed optimization of spatial queries
\end{itemize}

% \cleardoublepage
\includepublication{publications/fufezan-2009.pdf}
