\cleardoublepage
% ==============================================================
\chapter{Specific aims}
% ==============================================================

The vast amounts of data acquired in large-scale mass spectrometric experiments
require software systems which are powerful enough to perform data evaluation
in a way which is fast enough and does not represent a potential bottleneck in 
the experimental data evaluation workflow.
Therefore, the available systems which provide mass-spectrometric data 
evaluation platforms to researchers require further improvement.
Most importantly, a decentralized setup which supports all common types of 
operating systems can be expected to allow for increased and data processing 
throughput at minimal cost because commodity hardware can be employed.
Furthermore, the utilization of the various software tools needs to be further 
facilitated.
The described platform should be able to provide all necessary tools for
the evaluation of MS/MS data, including protein identification and quantitation.
The design of the system should be flexible and general enough to also
allow the incorporation of tools which are outside the scope of mass 
spectrometry, such as genomics or structural bioinformatics.

With such a decentralized MS/MS data evaluation system available, the
characterization of the chloroplast proteome in \cre~has been performed using
semi-quantitative analysis.
Furthermore, an analysis of the anaerobic response of the chloroplast proteome
was carried out, employing SILAC as a labeling strategy.
In order to carry out these analyses, specialized software for the 
high-throughput analysis of SILAC samples is required which is presented 
in this thesis.

The existing implementation of the Genomic Peptide Finder needs to
be improved in terms of search speed and sensitivity.
Another issue to be resolved is the statistical validation of GPF peptide 
assignments to MS/MS spectra.
Finally, the question arises whether peptides identified via {\em de novo} 
prediction and subsequent matching to the genome via the Genomic Peptide Finder 
can be used to establish a high-throughput proteogenomic genome annotation 
of \cre.
% It will be shown in this thesis that GPF-supported proteogenomic annotation 
% using high-throughput mass spectrometric data is feasible.
