\cleardoublepage
% ==============================================================
\chapter{Summary}
% ==============================================================

A novel platform for the evaluation of mass spectrometric data has been
established and was subsequently employed for the evaluation of measurements
performed to characterize the chloroplast proteome of \cre~and elucidate
the anaerobic response of the chloroplast proteome.
Proteomatic provides user-friendly access to mass spectrometric data 
evaluation programs and allows for the construction of complex workflows.

In order to facilitate the large-scale characterization of the chloroplast
proteome, qTrace has been designed to automate the peptide and protein 
quantitation process in full scans.
qTrace supports a variety of metabolic labeling strategies and has 
led to the confirmation of chloroplast protein induction under anaerobiosis
as well as the identification of several proteins of unknown function
which represent potential targets for further research.

{\em De novo}-based peptide sequencing is a powerful approach for the annotation
of MS/MS spectra, complementing gene model-based data search algorithms.
The inherent ambiguity of {\em de novo} sequenced peptides is greatly diminished
by the Genomic Peptide Finder which correlates the potentially erroneous
peptide sequences to the genomic DNA sequence of an organism, thereby discarding
spurious peptides and correcting ambiguous sequences.
An improved version of the Genomic Peptide Finder regarding sensitivity, 
specificity, and search speed, has been presented in this thesis.

Furhermore, a method for the automated statistical validation of GPF peptides
has been developed and has led to the possibility of using GPF peptides as
an extrinsic hint source for AUGUSTUS in a proteogenomic genome annotation 
approach.

All software developed in the scope of this thesis is publicly available
at \href{http://github.com/specht}{http://github.com/specht}.
